src/app/content/practiceQuestions/components/Answer.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import styled, { css } from 'styled-components/macro';
+import { h4Style } from '../../../components/Typography';
+import theme from '../../../theme';
+import ContentExcerpt from '../../components/ContentExcerpt';
+import { practiceQuestionStyles } from '../../constants';
+import { LinkedArchiveTreeSection } from '../../types';
+import { PracticeAnswer, PracticeQuestionStyles } from '../types';
+
+const correctAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'correct');
+const incorrectAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'incorrect');
+const selectedAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'selected');
+const unselectedAnswerStyle = practiceQuestionStyles.find((search) => search.label === 'unselected');
```

Comment:
you might as well defined these separately if the first thing you're gonna do is split them up